### ISSUE TITLE
Svg Export Problem

### ISSUE THREAD

**Issue by surgelata** [alexjlockwood/ShapeShifter#315]
Hi Guys, I'm sorry, I don't know if this is the right place for my question.

I'm using Shape Shifter, to create animation.
The problem is that after export my .svg with animation, the software generate several .svg from 0 to 40.
How can I have a single file with all the .svg to upload it on my wordpress and see the full animation?

Here the example:

[30fps.zip](https://github.com/alexjlockwood/ShapeShifter/files/2478008/30fps.zip)

Many thanks to all of you
Francesca

**Comment by alexjlockwood** [comment]
Yeah, so basically right now the only supported option for exporting for use on the web is w/ an SVG spritesheet. So basically Shape Shifter generates a frame-by-frame animation (w/ 40 frames in your case) and switches rapidly between them to create the animation. Kind of like a flipbook.

Shape Shifter should have also exported an `***_30fps.html` file that shows an example usage of how you could make use of this using HTML.

**Comment by acomito** [comment]
I really need the same thing. I was hoping it'd export a single SVG. That seems to be the industry standard from everything I've seen.

e.g. https://github.com/SamHerbert/SVG-Loaders/blob/master/svg-loaders/bars.svg

Is there anyway to get to the above manually from shapeshifter?
